src/Result.php

```diff
@@ -282,6 +282,25 @@ public function columnCount()
 
 
     /**
+     * Get an array containing only the specified column for each row.
+     *
+     * @param string $column The column to retrieve.
+     *
+     * @return array
+     */
+    public function getArrayOfColumn($column)
+    {
+        $result = $this->getValues();
+
+        if (!is_array($result)) {
```

Comment:
This line will always execute, as `getValues()` returns a generator, you've not tested this code